### PR TITLE
add

### PR DESCRIPTION
**Pull request by PirateBook1**

### CODE REVIEW

**Review by banned-historical-archives — APPROVED** [review]

**Comment by banned-historical-archives** [comment]
@PirateBook1 构建失败

**Comment by PirateBook1** [comment]
@banned-historical-archives 代码没问题啊

**Comment by banned-historical-archives** [comment]
@PirateBook1 https://github.com/banned-historical-archives/banned-historical-archives.github.io/actions/runs/4423100360

File: backend/parser/maoistlibrary/index.ts [hunk 1]
```diff
@@ -12,19 +12,30 @@ import {
 import { merge_to_lines, pdfjsContentToOCRResult } from '../utils';
 import { normalize } from '../../utils';
 import { join, basename } from 'node:path/posix';
-import peru from './peru';
-import india from './india';
-import philippines from './philippines';
-import turkey from './turkey';
+import Peru from './Peru';
```
Comment (banned-historical-archives) [inline backend/parser/maoistlibrary/index.ts:15]: 改成大写了

**Pull request merged**